custom/enikshay/integrations/utils.py

```diff
@@ -1,13 +1,14 @@
 from corehq.apps.locations.models import SQLLocation
-from custom.enikshay.exceptions import NikshayLocationNotFound
+from dimagi.utils.logging import notify_exception
 
 
 def is_submission_from_test_location(person_case):
     try:
         phi_location = SQLLocation.objects.get(location_id=person_case.owner_id)
     except SQLLocation.DoesNotExist:
-        raise NikshayLocationNotFound(
-            "Location with id {location_id} not found. This is the owner for person with id: {person_id}"
-            .format(location_id=person_case.owner_id, person_id=person_case.case_id)
-        )
+        message = ("Location with id {location_id} not found. This is the owner for person with id: {person_id}"
+                   .format(location_id=person_case.owner_id, person_id=person_case.case_id))
+        notify_exception(None, message="[ENIKSHAY] {}".format(message))
+        return True
```

Comment:
@mkangia what do you think of doing this here instead of failing hard here? I figured since you fall back to "yes" in the line below if the metadata isn't set, this might make sense too (if for some reason the location doesn't exist).